tap_mssql/sync_strategies/full_table.py

```diff
@@ -62,31 +81,99 @@ def sync_table(mssql_conn, config, catalog_entry, state, columns, stream_version
     ):
         singer.write_message(activate_version_message)
 
+    #TODO: Update code: If fastsync_batch_rows is not set, use old non fastsync method
     with mssql_conn.connect() as open_conn:
-        LOGGER.info("Generating select_sql")
-        select_sql = common.generate_select_sql(catalog_entry, columns)
-
         params = {}
 
         if catalog_entry.tap_stream_id == "dbo-InputMetadata":
             prev_converter = modify_ouput_converter(open_conn)
 
-        common.sync_query(
-            open_conn,
-            catalog_entry,
-            state,
-            select_sql,
-            columns,
-            stream_version,
-            table_stream,
-            params,
-        )
+        columns.sort()
+        select_sql = common.generate_select_sql(catalog_entry, columns, fastsync=True)
+ 
+        columns.extend(['_SDC_EXTRACTED_AT','_SDC_DELETED_AT','_SDC_BATCHED_AT'])
+        
+        query_df = df = pd.DataFrame(columns=columns) #TODO: delete?
```

Comment:
Since this works as is right now and will require rewriting/testing this, are you cool with leaving it as pandas and I can create another ticket in the R&R epic for updating this method?   If its best to get it switched to the most efficient asap, then i can make time this week to update this last part, but if its not pressing then i can do the ticket and get back to it.